build.rs

```diff
@@ -49,3 +50,18 @@ fn compile_sass() {
     let mut file = File::create(&dest_path).unwrap();
     file.write_all(css.as_bytes()).unwrap();
 }
+
+fn copy_js() {
+    use std::io::Read;
+    let mut javascript = String::new();
+    let source_path = Path::new(concat!(env!("CARGO_MANIFEST_DIR"), "/templates/menu.js"));
+    let dest_path = Path::new(&env::var("OUT_DIR").unwrap()).join("menu.js");
+    File::open(&source_path)
+        .expect("open templates/menu.js")
+        .read_to_string(&mut javascript)
+        .expect("read templates/menu.js");
+    File::create(&dest_path)
+        .expect("create target/.../menu.js")
+        .write_all(javascript.as_bytes())
+        .expect("write target/.../menu.js");
```

Comment:
nit: This might be easier using [`fs::copy`](https://doc.rust-lang.org/std/fs/fn.copy.html).